jobs/payment-jobs/tasks/stale_payment_task.py

```diff
@@ -99,38 +96,28 @@ def _delete_marked_payments(cls):
                 current_app.logger.warn(err)
 
     @classmethod
-    def _verify_recent_payments(cls):
+    def _verify_created_direct_pay_invoices(cls):
         """Verify recent invoice with PAYBC."""
-        recent_invoices = InvoiceModel.find_recent_cc_invoices(days=2)
-        if len(recent_invoices) == 0:
-            current_app.logger.info(f'Verify Invoice Job Ran at {datetime.datetime.now(tz=datetime.timezone.utc)}.'
+        created_invoices = InvoiceModel.find_created_direct_pay_invoices(days=2)
+        if len(created_invoices) == 0:
```

Comment:
`current_app.logger.info(f'Found {len(created_invoices)} Created Invoices to be Verified.')`